src/Model/SnapUser.hs

```diff
@@ -0,0 +1,28 @@
+module Model.SnapUser where
+  
+import Data.Time.Calendar
+import Snap.Snaplet.PostgresqlSimple
+
+data SnapUser = SnapUser
+  { id :: Int
+  , name :: String
+  , email :: String
+  , birthDate :: Day
+  , password :: String
+  , genre :: String
+  , income :: Double
```

Comment:
I know this is only an example, but normally we should avoid saving currency as floating point to represent currency. We should use a `Data.Decimal` (it is really precise) and store it in the database as a `Numeric`.